### DISCUSSION TITLE
Creating a keymap for moving to next and previous Symbol

### DISCUSSION THREAD

**Discussion by MatthewAlbrecht** [zed-industries/zed#17038] [Feature Requests]
I want to have a keymap for moving between the next and previous symbol.  I've tried two approaches and have hit dead ends:

1. A built in `Editor::NextSymbol`
2. Using the OutlinePanel and `workspace::SendKeystrokes` to toggle focus then use arrow keys to go down one and then press enter, but enter doesn't select the symbol, only a mouse click does.

Any thoughts? Am I just missing something simple here?

**Comment by Kasparsu** [comment]
idk maybe they updated or something I figured out this 

```json
 {
    "context": "Editor",

    "bindings": {
      "ctrl-t": "outline::Toggle",
      "ctrl-1": ["workspace::SendKeystrokes", "ctrl-t down enter"]
    }
  }
```

I wanted to rebind symbols list from ctrl+shift+o to ctrl+t so thats here as well but ctrl+1 in this example moves to next symbol I would guess you could do this in reverse for previous one as well.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
